tools/android_lint/bin/main.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+/// Runs the Android SDK Lint tool on flutter/shell/platform/android.
+///
+/// This script scans the flutter/shell/platform/android directory for Java
+/// files to build a `project.xml` file.  This file is then passed to the lint
```

Comment:
Where are the lint rules that are being applied?

Reply:
Right now all of the default ones and all warnings, which are all getting elevated to errors.

Reply:
But the linter includes style doesn't it?  If so, which of the many flavors of java/Android style is it applying?

Reply:
I'm also curious where these are actually defined.  I don't know any default lint rules off the top of my head.

Reply:
This is the public doc for it: https://developer.android.com/studio/write/lint

I don't know that it actually checks what we'd be calling style - or certainly not everything in the Java style guide.